### DISCUSSION TITLE
Is there a way to get a static "latest" version msi link?

### DISCUSSION THREAD

**Discussion by mikedoy** [microsoft/terminal#12503] [General]
Instead of : 
pseudo code:
latest_ver = curl -sI https://github.com/microsoft/terminal/releases/latest | grep location:
version := path.Base(latest_ver)
versionNoV := version[1:]
wtfisthis := "8wekyb3d8bbwe"
base_url = "https://github.com/microsoft/terminal/releases/download"
msi_url = fmt.Sprintf("%s/%s/Microsoft.WindowsTerminal_%s_%s.msixbundle", base_url, version, versionNoV, wtfisthis)

**Comment by Makishima** [comment]
@mikedoy try to fetch `curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/microsoft/terminal/releases/latest`
and then find value of the key `browser_download_url` based on `"content_type": "application/octet-stream"`

**Reply by mikedoy** [reply]
That helps. Thanks!
